metadata/Cadastre/basic_parcels/rich_county_parcels_schema.md

```diff
@@ -0,0 +1,11 @@
+# Title
+
+Rich County Parcels Field Definitions
+
+## ID
+
+4757d79b-8591-4693-99b9-7e83f0205db8
+
+### NOTES
```

Comment:
Rich does not have a unique field _NOTES_

Reply:
https://services1.arcgis.com/99lidPhWCzftIe9K/arcgis/rest/services/Parcels_Rich/FeatureServer/0

![image](https://github.com/user-attachments/assets/08cb6a3a-9b32-4cf8-a4b6-900bbcba6dd3)

It does have that field and they are all empty.